### ISSUE TITLE
application decompile not work .. java.lang.NullPointerException

### ISSUE THREAD

**Issue by TechComet** [Konloch/bytecode-viewer#327]
Hi,

just open the app
```
java.lang.NullPointerException
        at the.bytecode.club.bootloader.Boot.setState(Boot.java:269)
        at the.bytecode.club.bootloader.Boot.dropKrakatau(Boot.java:310)
        at the.bytecode.club.bytecodeviewer.util.InstallFatJar.run(InstallFatJar.java:42)
        at java.lang.Thread.run(Thread.java:748)
```

and in `file` menu all `save and decompile` not working

**Comment by ThexXTURBOXx** [comment]
This bug was already fixed on `master`. Please try with this release: https://femtopedia.de/bytecodeviewer-2.10.14.jar

**Comment by Konloch** [comment]
Thank you for the report! To help fix this issue I've pushed [v2.10.14](https://github.com/Konloch/bytecode-viewer/releases/tag/v2.10.14)
